resources/assets/lib/beatmap-discussions/event.tsx

```diff
@@ -190,6 +202,11 @@ export default class Event extends React.PureComponent<Props> {
       params.beatmap = osu.link(route('beatmaps.show', { beatmap: data.beatmap_id }), data.beatmap_version);
     }
 
+    if (isNominationResetReceivedEventJson(this.props.event)) {
+      const data = this.props.event.comment;
+      params.user = osu.link(route('users.show', { user: data.source_user_id }), data.source_user_username);
```

Comment:
can use information on whose nomination is being reset instead of just a bunch of "Nomination was reset by user (reason)" for the listing page